### PR TITLE
Frontend: attach categories tags.

### PR DESCRIPTION
**Pull request by vlvereta**

### CODE REVIEW

File: frontend/src/components/place/AddPlace.vue [hunk 1]
```diff
@@ -394,6 +378,10 @@ export default {
         'newPlace.category': function (categoryObject) {
             if (categoryObject) {
```
Comment (nemoipaha) [inline frontend/src/components/place/AddPlace.vue]: Better to write logic vie versa here
if (_.isEmpty(categoryObject)) { return;  }
... // logic

File: frontend/src/store/modules/place/category/actions.js [hunk 2]
```diff
@@ -25,5 +25,17 @@ export default {
                     reject(error);
                 });
         });
+    },
+
+    getTagsByCategory: (context, categoryId) => {
+        return new Promise((resolve, reject) => {
+            httpService.get('/places/categories/' + categoryId + '/tags')
```
Comment (nemoipaha) [inline frontend/src/store/modules/place/category/actions.js]: Use `/places/categories/${categoryId}/tags` a bit easier )

File: frontend/src/store/modules/place/category/actions.js [hunk 3]
```diff
@@ -25,5 +25,17 @@ export default {
...
+                .then(function (result) {
```
Comment (nemoipaha) [inline frontend/src/store/modules/place/category/actions.js]: Use () => {}

File: frontend/src/store/modules/place/category/actions.js (hunk 2 above)
Comment (vlvereta) [reply frontend/src/store/modules/place/category/actions.js]: It was backquotes..
thx

**Pull request merged**